packages/autocomplete-theme-classic/src/theme.scss

```diff
@@ -1,229 +1,471 @@
+                }
+            }
+        }
+    }
 }
 
-.aa-Label,
-.aa-TouchSearchButtonIcon {
-  align-items: center;
-  color: #777;
-  cursor: initial;
-  display: flex;
-  height: 100%;
+// ----------------
+// Panel
+// ----------------
+.aa-Panel {
+    position: absolute;
+    //reset
+    * {
+        box-sizing: border-box;
+        margin: 0;
+        padding: 0;
+    }
+    button {
+        appearance: none;
+        background: none;
+        border: none;
+    }
+    &.aa-Panel--desktop {
+        .aa-PanelLayout {
+            //display: flex;
+            margin-top: var(--aa-spacing-half);
+            padding-top: var(--aa-spacing-half);
+            border-radius: 3px;
+            background-color: var(--aa-background-color);
+            box-shadow: var(--aa-panel-shadow);
+            text-align: left;
+            padding-bottom: var(--aa-spacing-half);
+            overflow: hidden;
+            .col-results {
+                // max-height: 400px;
+                width: 50%;
+                overflow-y: scroll;
+                // section.aa-Source
+            }
+            aside.col-preview {
+                //max-height: 400px;
+                //animation: fadeIn ease-in .2s;
+                flex-shrink: 1;
+                max-width: 50%;
+                overflow: hidden;
+                border-left: solid 1px var(--aa-selected-color);
+                &:empty {
+                    //display:none;
+                }
+            }
+        }
+    }
+    &.aa-Panel--touch {
+        right: var(--aa-spacing-half);
+        left: var(--aa-spacing-half);
+        padding-top: var(--aa-spacing-half);
+        .aa-Item {
+            padding: 0;
+        }
+        .aa-SourceHeader {
+            margin: var(--aa-spacing-half) 0 var(--aa-spacing-half) 2px;
+        }
+    }
+    // when a query isn't resolved yet
+    &.aa-Panel--stalled {
+        filter: grayscale(1);
+        opacity: 0.5;
+        transition: opacity 200ms ease-in;
+    }
 }
 
-.aa-InputWrapper,
-.aa-TouchSearchButton {
-  align-items: center;
-  background-color: #fff;
-  display: flex;
-  height: 2.5rem;
-  position: relative;
-  width: 100%;
+// ----------------
+// Sources
+// children of .col-results
+// Each section can be styled independantly here
+// ----------------
+section.aa-Source {
+    width: 100%;
+    // source title
+    .aa-SourceHeader {
+        position: relative;
+        line-height: var(--aa-spacing);
+        margin: var(--aa-spacing-half) var(--aa-spacing);
+        // title typography
+        span {
+            color: var(--aa-primary-color);
+            display: inline-block;
+            position: relative;
+            z-index: 2;
+            padding-right: var(--aa-spacing-half);
+            font-size: 0.8em;
+            font-weight: 600;
+            text-transform: capitalize;
+            background: var(--aa-background-color);
+        }
+        // Line separator
+        .aa-SourceHeaderLine {
```

Comment:
Could we rely on pseudo-elements and not have to insert a DOM element to get this style? If users have to think about this element, they'll likely not use it.

Reply:
I think it's easier to remove the line if you don't want to use it